docs/quickstarts/expo.mdx

```diff
@@ -666,6 +666,6 @@ Now that you have an application integrated with Clerk, you will want to read th
 
     <Cards title="Authentication Components" description="Learn more about all our authentication components." link="/docs/components/authentication/sign-in" cta="Learn More" />
 
-    <Cards title="Client Side Helpers" description="Learn more about our client side helpers and how to use them." link="/docs/references/react/use-auth" cta="Learn More" />
+    <Cards title="Client Side Helpers" description="Learn more about our client side helpers and how to use them." link="/docs/references/react/use-user" cta="Learn More" />
```

Comment:
The first hook on React's list of hooks is useUser(), so let's link out to that one so that it's straightforward.